lib/elasticity/search.rb

```diff
@@ -118,6 +118,26 @@ def search_results
         end
       end
 
+      # for pagination
+      def per_page
+        @search_definition.body[:size] || 10
```

Comment:
This shouldn't default to a hardcoded 10, make it `response["hits"]["hits"].length`